Microsoft.HealthVault/Connection/SessionCredential.cs

```diff
@@ -20,5 +22,13 @@ public class SessionCredential
         /// Gets or sets the shared secret.
         /// </summary>
         public string SharedSecret { get; set; }
+
+        /// <summary>
+        /// Gets or sets the created time of the token.
+        /// </summary>
+        /// <value>
+        /// The created.
+        /// </value>
+        public DateTime ExpirationUtc { get; set; }
```

Comment:
Updated to DateTimeOffset for consistency 

---
In reply to: [115837766](https://github.com/Microsoft/healthvault-dotnetstandard-sdk/pull/27#discussion_r115837766) [](ancestors = 115837766,115805292,115796836)